imager_profile/models.py

```diff
@@ -24,76 +24,62 @@ class Meta:
         model = "PatronProfile"
 
     def get_queryset(self):
-        qs = super(PatronProfileManager, self).get_queryset()
-        return qs.filter(user__is_active=True)
+        '''retrn a queryset of active users'''
+        return User.objects.filter(is_active=True)
 
 
 @python_2_unicode_compatible
 class Photographer(models.Model):
     user_uuid = models.UUIDField(primary_key=True,
                                  default=uuid.uuid4,
-                                 editable=False)
+                                 editable=False,
+                                 )
     user = models.OneToOneField(settings.AUTH_USER_MODEL,
-                                on_delete=models.CASCADE)
-    has_portfolios = models.BooleanField(default=False)
-    portfolio_url = models.CharField(max_length=255, blank=True)
+                                on_delete=models.CASCADE,
+                                related_name='photographer',)
+    bio = models.CharField('Bio',
+                           max_length=1024,
+                           blank=True)
 
     def __str__(self):
-        fn = self.user.get_full_name().strip() or self.user.get_username()
-        return "{}: {}".format(fn, self.user_uuid)
-
-    @property
-    def is_active(self):
-        return self.user.is_active
+        return self.user.get_full_name() or self.user.username
 
     objects = models.Manager()
     active = PatronProfileManager()
 
 
 class Address(models.Model):
-    photographer_profile = models.ForeignKey(
-        Photographer,
-        on_delete=models.CASCADE,
-        primary_key=True,
-        related_name='Addresses',  # TODO: look this up and set it
-    )
+    photographer = models.ForeignKey(settings.AUTH_USER_MODEL,
```

Comment:
This is quite confusing, since this is a relationship to the `User`model, not the `Photographer` model, while the field is called `photographer`.